projects/plugins/jetpack/extensions/blocks/ai-image/index.js

```diff
@@ -30,6 +30,12 @@ export const settings = {
 					'jetpack'
 				) }
 			</p>
+			<p>
+				{ __(
+					'The content is generated by OpenAI. Jetpack does not endorse, nor vouch for accuracy of the output. Please use your best judgement as you are the ultimate arbiter of what you publish on your site.',
```

Comment:
Would the first sentence be better something like the following (to provide more context for people who don't know what OpenAI is)?

>The content is generated by a computer using OpenAI and has not been reviewed by a human.